src/components/content/content.scss

```diff
@@ -53,6 +53,13 @@ ion-content.js-scroll > .scroll-content {
   will-change: initial;
 }
 
+ion-content[no-scroll] > .scroll-content {
+  overflow-x: initial;
+  overflow-y: initial;
+  -webkit-overflow-scrolling: auto;
+  will-change: initial;
```

Comment:
@AmitMY I used this as `ion-content.js-scroll > .scroll-content` uses it as well, so I figured there might be a reason. Wouldn't matter much anyway as its contained elsewhere as well. Will change it for both if requested!